packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/process_termination_details.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// This class provides more specific information about why the render process exited.
+/// The application may use this to decide how to handle the situation.
+@immutable
+class ProcessTerminationDetails {
+  /// Creates a [ProcessTerminationDetails].
+  const ProcessTerminationDetails({
+    required this.didCrash,
+    required this.rendererPriorityAtExit,
+  });
+
+  /// Indicates whether the render process was observed to crash, or whether it was killed by the system.
+  final bool didCrash;
```

Comment:
What would a platform implementation that didn't have access to this information pass? It seems like this needs to be an enum, where one of the options is "unknown".